apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts

```diff
@@ -2,6 +2,8 @@ import { IsArray, IsString } from 'class-validator'
 import { ApiProperty } from '@nestjs/swagger'
 import { Type } from 'class-transformer'
 
+console.log('-trigger schemas')
```

Comment:
remove this?

Reply:
Still work in progress, the file is not detected when it is updated and doesn't trigger cache/hash-key zip